posthog/api/action.py

```diff
@@ -284,7 +279,3 @@ def count(self, request: request.Request, **kwargs) -> Response:
             },
         )
         return Response({"count": results[0][0]})
-
-
-class LegacyActionViewSet(ActionViewSet):
```

Comment:
This was the only "legacy" viewset I could find that isn't actually referenced anywhere so I removed it